pkg/minikube/cluster/cluster_darwin.go

```diff
@@ -60,7 +60,6 @@ func createXhyveHost(config MachineConfig) *xhyveDriver {
 		Memory:         config.Memory,
 		CPU:            config.CPUs,
 		Boot2DockerURL: config.GetISOFileURI(),
-		BootCmd:        "loglevel=3 user=docker console=ttyS0 console=tty0 noembed nomodeset norestore waitusb=10 base host=" + constants.MachineName,
```

Comment:
We don't need it anymore because same options not works for our iso and latest xhyve driver already have required fix for same.